### PR TITLE
UN-646: Featured record article

### PR DESCRIPTION
**Pull request by nicklee**
Ticket URL: https://national-archives.atlassian.net/jira/software/c/projects/UN/boards/75?modal=detail&selectedIssue=UN-646
## About these changes

Adding record revealed component to the highlight gallery page

## How to check these changes

Add a record to the highlight gallery page

Screenshot 2023-05-24 at 17 13 06

## Before assigning to reviewer, please make sure you have

- [ ] Checked things thoroughly before handing over to reviewer.
- [ ] Checked PR title starts with ticket number as per project conventions to help us keep track of changes.
- [ ] Ensured that PR includes only commits relevant to the ticket.
- [ ] Waited for all CI jobs to pass before requesting a review.
- [ ] Added/updated tests and documentation where relevant.

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

File: templates/includes/record-revealed.html [hunk 1]
```diff
@@ -1,21 +1,34 @@
...
+                    height="{{ mobile_img.height }}"
+                    width="{{ mobile_img.width }}"
+                    src="{{ mobile_img.url }}"
+                    alt=""/>
+            </picture>
+        </a>
+    {% endif %}
+    <div class="featured-record-article__container">
+        <p class="featured-record-article__meta u-margin-xxs">{% trans "Record Revealed" %}</p>
+        <a href="{{ record.record.url }}" class="featured-record-article__link">
```
Comment (William-Blackie) [inline templates/includes/record-revealed.html:24]: Note: Looking at tabbing for this element this link does not have an `active` psudeo state. Could we add one as it's not clear for our screen reader users.

Comment (William-Blackie) [reply templates/includes/record-revealed.html:24]: Also, it looks like the typography styles is overidding our link colour here; 
Screenshot from 2023-05-25 12-02-48

We might need a more specific selector here to override it, or removing that blanket style which would be my preference as we mostly just override it anyway.

File: templates/includes/record-revealed.html [hunk 2]
```diff
@@ -1,21 +1,34 @@
...
+{% load i18n wagtailimages_tags wagtailcore_tags records_tags %}
+{# TODO this is a near duplicate of template/blocks/featured_record_article.html #}
+
+<div class="featured-record-article featured-record-article--standalone"
+    data-container-name="featured-record-article">
+    {% if record.teaser_image %}
+        <a href="{{ record.record.url }}">
+            <span class="sr-only">{{ record.title }}</span>
+            <picture>
+                {% image record.teaser_image fill-626x486-c100 as mobile_img %}
```
Comment (William-Blackie) [inline templates/includes/record-revealed.html:10]: Note: Looking at the designs again, I think we've made this image slighty smaller in width, potentially worth double checking while we're here.

**Review by William-Blackie — DISMISSED** [review]
Hey @nicklee, this looks good, just some suggestions / double checking. Thanks!

File: templates/includes/record-revealed.html [hunk 3]
```diff
@@ -1,21 +1,34 @@
...
-            </div>
-            <div class="col-4 explore-intro__image-container">
-                {% if page.intro_image %}
-                    {% image page.intro_image fill-390x390 as intro_image %}
-                    <img height=390 width=390 src="{{ intro_image.url }}" alt="" class="explore-intro__image" />
-                {% endif %}
-            </div>
-        </div>
+{% load i18n wagtailimages_tags wagtailcore_tags records_tags %}
+{# TODO this is a near duplicate of template/blocks/featured_record_article.html #}
```
[the remark this answers is not in the document]
Comment (nicklee) [reply templates/includes/record-revealed.html]: Thanks, I've fixed this so we're now using the include as the main component, with the block referencing the include

**Review by jamesbiggs — CHANGES_REQUESTED** [review]
Hi @nicklee, there are a few errors in the `_highlight-gallery.scss` that are preventing it from running - I've added a commit that seems to fix it - if you wouldn't mind double checking it and making sure I've not missed anything out/deleted anything that was needed 😃 

I also wrapped it in a div with the `container` class to give it a bit of padding as it's currently stretching further than other page components are.

**Review by janehughes — APPROVED** [review]
This seems good to go to me - I've added one tiny change to vertically center the content in the featured record

**Review by jamesbiggs — APPROVED** [review]

**Pull request merged**